### ISSUE TITLE
ethgasstation.info has been retired

### ISSUE THREAD

**Issue by Rachel-Eichenberger** [Uniswap/smart-order-router#277]
- **I'm submitting a ...**
  [X ] bug report
  [ ] feature request
  [ ] question about the decisions made in the repository
  [ ] question about how to use this project

- **Summary**

As a backup for gas information when not on RPC we have `https://ethgasstation.info/` as a fallback, This service has be retired, and the code in line https://github.com/Uniswap/smart-order-router/blob/2cd43f1174c91740c1ae09fae8fcd8d6773d2df7/src/routers/alpha-router/alpha-router.ts#L579 will have to be updated to a new service if available.

**Comment by robertmclaws** [comment]
Yes, can we please get this fixed ASAP? It's polluting the console with error messages.

**Comment by jsy1218** [comment]
We might have to use tenderly (which is already built-in) instead of [smart-order-router](https://github.com/Uniswap/smart-order-router) its own gas heuristic or any other gas provider in the long term. Gas estimate has become a critical issue lately since UniswapX launch.

We will use tenderly with the perf optimization.

**Comment by robertmclaws** [comment]
Looking forward to a fix for this and #396!
